Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14 as builder
+FROM node:lts-gallium as builder
```

Comment:
@war1oc 
Can you actually use the v16 tag instead of a named tag?

Reply:
@yashmurty We specifically want to target the LTS version, not just the v16.
This only changes the meaning semantically, but tools like dependabot will behave differently depending on this.

Reply:
@Seb-C 
Got it!
Thanks for sharing.

@war1oc shared about this in today's node.js meeting with me.
Makes sense. 
I have asked Tanveer to add a comment on top explaning `gallium` is the name for v16.